model_manager/src/transformers/BaseTransformer.py

```diff
@@ -2,21 +2,21 @@
 
 class BaseTransformer:
     @abstractmethod
-    def __init__(self, config):
+    def __init__(self, config: dict):
         """
-        config: dict passed from the pv_mappings.yaml files
+        config: dict passed from the pv_mappings.yaml files.
         """
         pass
     
     @abstractmethod
     def transform(self):
         """
-        Initiate transform function to transform the input data
+        Call transform function to transform the input data, see SimpleTransformer in model_manager/src/transformers/BaseTransformers.py for an example.
         """
         pass
     
     @abstractmethod
-    def handler(self, pv_name, value):
```

Comment:
Add description of what the objects are:
* pv_name and the value associated with that PV
* this comes from me being ignorant but currently it cant handle arrays?